api-update/common/src/test/resources/json/graphql_valid_av_multiple_records_multiple_expected.json

```diff
@@ -0,0 +1,25 @@
+{
+  "query" : "mutation AddAntivirusMetadata($input:[AddAntivirusMetadataInput!]!){addAntivirusMetadata(addAntivirusMetadataInput:$input){fileId software value softwareVersion databaseVersion result datetime}}",
```

Comment:
Do we need to support being able to updating antivirus results for multiple files at once? Would it be simpler to just remove that functionality?

Reply:
Yeah we do. Lambdas can receive more than one message sqs message per invocation so it's likely that the lambda will be updating more than one file and this cuts down on the number of api requests.

Reply:
Is that a built-in feature of connecting SQS to a Lambda, or is that something we've decided to support? I'm just trying to think through the implications of that.

Reply:
It's a built in feature.

Reply:
I think you can set the batch size to 1 which will run 1 lambda per message but we might later run into issues with lambda limits and this code will still work with 1 lambda per message. This gives us the option to change the batching size in the future with a single terraform parameter whereas if it's written for a single record and decide to batch stuff after testing it, we'd have to re-write all of this.